formats/video.md

```diff
@@ -11,21 +11,38 @@ Welcome to Prebid video ad support. This is a collection of resources covering
 how Prebid can help you monetize video. If you're new to Prebid video, a
 good place to start would be the [Prebid.js-focused training video overview](/prebid-video/video-overview-video.html).
 
-## Instream
+## Mapping Out Prebid Integration with the IAB Use Cases/Video Placement Types
 
-Instream ads are for short-form video content within a player generally has simple video ad requirements, e.g. a single pre-roll ad.
+The IAB has identified the [four types of video placements](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/main/AdCOM%20v1.0%20FINAL.md#list_plcmtsubtypesvideo) for the introduced `plcmt` attribute in `Object:Video`, in accordance with the IAB Digital Video Guidelines: 
 
-1. [Prebid.js instream video overview](/prebid-video/video-overview.html#instream-video)
-1. [Prebid.js: Getting started with Prebid video](/prebid-video/video-getting-started.html)
-1. [Prebid Server video ad support](/prebid-server/use-cases/pbs-pbjs.html)
+1. **Instream:** pre-roll, mid-roll and post-roll ads that are played before, during or after the streaming video content, which the viewer has requested. Instream video must be set to “sound on” by default at player start, or have explicitly clear user intent to watch the video content. While there may be other content surrounding the player, the video content must be the focus of the user’s visit.
 
-## Outstream
+2. **Accompanying Content:** the video player loads and plays before, between, or after paragraphs of text or graphical content.
 
-'Outstream' ads are packaged with a separate player, or the publisher provides a special player for them. They aren't embedded within video content, but rather exist as standalone video players somewhere on the page.
+3. **Interstitial:** video ads that are played without video content. During playback, the video ad must be the primary focus of the page, take up the majority of the viewport and cannot be scrolled out of view.
 
-1. [Prebid.js outstream video overview](/prebid-video/video-overview.html#outstream-video)
-1. [Prebid.js: Getting started with Prebid video](/prebid-video/video-getting-started.html)
-1. [Prebid Server video ad support](/prebid-server/use-cases/pbs-pbjs.html)
+4. **No Content/Standalone:** video ads that are played without the streaming video content.
+
+However, for the described four use cases/placement types, there are only two types of integration from the Prebid perspective:
+
+1. [**In-player.**](https://docs.prebid.org/prebid-video/video-overview#in-player-integration) There's already a video player on the page. The publisher has to use the [Video Module](https://docs.prebid.org/prebid-video/video-module.html) or implement a javascript function that passes bid responses to the player. In this scenario, Prebid handles video bid caching server-side, then maps it to a unique cache ID, which will be passed to the ad server via key-value targeting. 
```

Comment:
```suggestion
1. [**In-player.**](/prebid-video/video-overview#in-player-integration) There's already a video player on the page. The publisher has to use the [Video Module](https://docs.prebid.org/prebid-video/video-module.html) or implement a javascript function that passes bid responses to the player. In this scenario, Prebid handles video bid caching server-side, then maps it to a unique cache ID, which will be passed to the ad server via key-value targeting. 
```

Using relative links is usually better, because

1. navigating on preview deployments works correctly (you don't navigate to the live site)
2. detection of broken links work, because Jekyll can lookup the file - if there's no file, it's a broken link
3. you get auto completion in the editor for files